src/Middleware/Zed/RabbitMqProcess/Communication/Console/RabbitMqProcessMessagePublisherConsole.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Middleware\Zed\RabbitMqProcess\Communication\Console;
+
+use Spryker\Zed\Kernel\Communication\Console\Console;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @method \Middleware\Zed\RabbitMqProcess\Business\RabbitMqProcessFacadeInterface getFacade()
+ */
+class RabbitMqProcessMessagePublisherConsole extends Console
+{
+    protected const COMMAND_NAME = 'queue:publish-messages';
+    protected const DESCRIPTION = 'Publishes messages to queue';
+    protected const OPTION_MESSAGES_AMOUNT = 'amount';
+    protected const OPTION_MESSAGES_AMOUNT_SHORTCUT = 'a';
+
+    /**
+     * @return void
+     */
+    protected function configure(): void
+    {
+        $this->setName(static::COMMAND_NAME)
+            ->setDescription(static::DESCRIPTION);
+
+        $this->addOption(
```

Comment:
Options are optional and value can be required `InputOption::VALUE_REQUIRED` but in this case, the option is not optional, it will throw an exception because `publishMessages` expects a value, if you want to keep it as an option, you have to add a default value or make it an argument.